### DISCUSSION TITLE
redacting locality info for rare species syns and subtaxa

### DISCUSSION THREAD

**Discussion by Tambuyukon** [Symbiota/community-board#131] [Q&A] [answered]
Greetings,

This probably should be assigned to another category at some point (wishlist or problem/bugs)?

Framework within Symbiota does not currently support redaction of locality info for synonyms or subtaxa of species that appear on rare species lists. How do we accomplish this? For example a search for Aster georgianus from South Carolina will display specimens assigned to this name AND Symphytrichum georgianum. The accepted name S. georgianum is on the Rare Species of SC list and locality details and images are redacted for these records. However, even though A. georgianus is linked as synonym of S. georgianum in the taxon tree, the records entered with current determinations of A. georgianus from SC show all locality details and images. The same occurs if you have a subspecies or variety of a Rare Species. One 'clunky' workaround is to just add all of the synonyms to the rare species list, but this again essentially makes nomenclators out of curators.

Cheers,
T

**Comment by sginzbar** [comment]
I went though the list of federally listed plants at https://ecos.fws.gov/ecp/report/species-listings-by-tax-group?statusCategory=Listed&groupName=All%20Plants. Of 167 taxa in region 4 (southeastern US and Puerto Rico) only 25 show some locality details and images in a SERNEC specimen search of all herbaria including synonyms:

Trichomanes punctatum ssp. floridanum
Baptisia arachnifera
Chamaecrista lineata keyensis
Chamaesyce deltoidea pinetorum
Chamaesyce deltoidea serpyllum
Chamaesyce deltoidea ssp. deltoidea
Dalea carthagenensis floridana
Daphnopsis helleriana
Dicerandra immaculata
Digitaria pauciflora
Harrisia (=Cereus) aboriginum   (=gracilis)
Juglans jamaicensis
Leavenworthia crassa
Liatris helleri
Linum arenicola
Linum carteri carteri
Paronychia chartacea
Physaria globosa
Pilosocereus robinii
Platanthera integrilabia
Sarracenia rubra ssp. alabamensis
Sideroxylon reclinatum ssp. austrofloridense
Spigelia gentianoides
Stahlia monosperma
Trillium reliquum

Thirteen others returned no records so I couldn't tell if the locality information is protected or not.

I am attaching a spreadsheet with protected and unprotected, names, varieties and subspecies and synonyms. (It was sometimes impossible to tell why some specimens of the same taxon and from the same herbarium were protected and others not,)

Steve

[Listed Plant Taxa Region 4 - Localities not protected.xlsx](https://github.com/BioKIC/symbiota-docs/files/8356104/Listed.Plant.Taxa.Region.4.-.Localities.not.protected.xlsx)

**Comment by sginzbar** [comment]
[Listed.Plant.Taxa.Region.4.-.Localities.not.protected.xlsx](https://github.com/BioKIC/symbiota-docs/files/8356407/Listed.Plant.Taxa.Region.4.-.Localities.not.protected.xlsx)

Same table. Headers on top.

Steve

**Comment by themerekat** [comment, marked as answer]
Hi @Tambuyukon and @sginzbar 
Very good points, thanks for bringing this up! I did some testing and confirmed that non-accepted synonyms do not automatically inherit the same locality protections as their accepted synonyms.
As for Symphytrichum georgianum, this taxon is currently not listed as requiring locality protections.
Redacting locality data for all protected species is not necessarily a universally adopted practice and must be decided by portal communities. If you wish to redact these data for your own collection(s) immediately, you can individually protect the localities using the locality security checkbox (https://biokic.github.io/symbiota-docs/coll_manager/data_publishing/redaction/) or send us a list of the taxa or specimens you would like to have redacted for your collection.
